pkg/known/const.go

```diff
@@ -4,3 +4,8 @@ const (
 	CraneSystemNamespace = "crane-system"
 	AutoscalingFinalizer = "autoscaling.crane.io/finalizer"
```

Comment:
this comment was missed in last pr, if we use ownerRef, seems this finalizer is not required?

Reply:
the same above